client/hotrod-client/src/main/java/org/infinispan/client/hotrod/impl/RemoteCacheImpl.java

```diff
@@ -348,6 +349,13 @@ public boolean containsKey(Object key) {
    }
 
    @Override
+   public boolean containsValue(Object value) {
+      Objects.requireNonNull(value);
+      // TODO: This would be more efficient if done on the server itself and requested via protocol
```

Comment:
Yes, actually this and below are essentially the same thing. I thought I had refactored this one to call the other, let me do that and I will just put the TODO on the bottom one.